containerimages/container-disk-images/create-push-vm-image.sh

```diff
@@ -0,0 +1,60 @@
+#!/usr/bin/env bash
+set -ex
+
+SCRIPT_DIR="$(
+    cd "$(dirname "$BASH_SOURCE[0]")"
+    pwd
+)"
+
+export REGISTRY=${REGISTRY:-quay.io}
+export REPOSITORY=${REPOSITORY:-omergi}
+export IMAGE_NAME=${IMAGE_NAME:-fedora-extended}
+export TAG=${TAG:-tests}
+
+readonly VM_IMAGE_URL="https://download.fedoraproject.org/pub/fedora/linux/releases/32/Cloud/x86_64/images/Fedora-Cloud-Base-32-1.6.x86_64.qcow2"
+readonly VM_IMAGE="source-image.qcow2"
+
+full_image_tag="$REGISTRY/$REPOSITORY/$IMAGE_NAME/$TAG"
+build_directory="${IMAGE_NAME}_build"
+new_vm_image_name="provisioned-image.qcow2"
+
+trap 'cleanup' EXIT
+
+function cleanup() {
+    echo "cleanup"
+    rm -rf $build_directory || true
+}
+
+function customize_image() {
+  local source_image=$1
+  local customized_image=$2
+
+  vm_image_temp="temp-${source_image}"
+
+  # Backup the VM image and pass copy of the original image
+  # in case customizing script fail.
+  cp $source_image "$vm_image_temp"
+
+  # TODO: convert this script to container
+  ${SCRIPT_DIR}/customize-image.sh "$vm_image_temp" "$customized_image"
+
+  # Backup no longer needed.
+  rm -f $vm_image_temp
+}
+
+pushd $SCRIPT_DIR
```

Comment:
no need for ident, and you can use '(' and ')' and then no need pushd and popd as it will run in a subshell, and the pwd outside will stay the same, and then it will be good to use ident please